script.js

```diff
@@ -24,6 +14,121 @@ var data = [
   { label: '3x Jägerbombs'      , value: 1, xp: '12' },
   { label: '2x Corona' , value: 1, xp: '13' },
 ];
+
+var mixers = [
+  { label: '1x Coca Cola', value: 1, xp: '2' },
+  { label: '1x Sparkling water', value: 1, xp: '4', },
+  { label: '1x Tonic Water', value: 1, xp: '1', },
+  { label: '1x Ginger Beer', value: 1, xp: '4', },
+  { label: '1x Pineapple Juice', value: 1, xp: '5' },
+  { label: '1x Orange Juice', value: 1, xp: '3' },
+  { label: '1x Red Soda', value: 1, xp: '9' },
+  { label: '1x Coca Cola', value: 1, xp: '10' },
+  { label: '1x Tonic Water', value: 1, xp: '10' },
+  { label: '1x Ginger Beer', value: 1, xp: '11' },
+  { label: '1x Pineapple Juice', value: 1, xp: '12' },
+  { label: '1x Orange Juice', value: 1, xp: '13' },
+];
+
+var shots = [
+  { label: '2x Tequila', value: 1, xp: '2' },
+  { label: '1x Jägermeister', value: 1, xp: '4', },
+  { label: '1x Sambuca', value: 1, xp: '1', },
+  { label: '1x Pure Apple', value: 1, xp: '5' },
+  { label: '1x Pure Liquorice', value: 1, xp: '5' },
+  { label: '1x Råstof Caramel', value: 1, xp: '5' },
+  { label: '1x Råstof Rhubarb', value: 1, xp: '5' },
+  { label: '1x Rum', value: 1, xp: '3' },
+  { label: '1x Gin', value: 1, xp: '9' },
+  { label: '2x Fernet', value: 1, xp: '10' },
+  { label: '2x Mintuu', value: 1, xp: '12' },
+  { label: '1x Fireball', value: 1, xp: '13' },
+]
+
+var sirup = [
+  { label: '1x Mango', value: 1, xp: '2' },
+  { label: '1x Passionfruit', value: 1, xp: '2' },
+  { label: '1x Lime', value: 1, xp: '2' },
+  { label: '1x Sugar', value: 1, xp: '2' },
+  { label: '1x Grenadine', value: 1, xp: '2' },
+  { label: '2x Mango', value: 1, xp: '2' },
+  { label: '2x Passionfruit', value: 1, xp: '2' },
+  { label: '2x Lime', value: 1, xp: '2' },
+  { label: '2x Sugar', value: 1, xp: '2' },
+  { label: '2x Grenadine', value: 1, xp: '2' },
+  { label: '3x Mango', value: 1, xp: '2' },
+  { label: '3x Passionfruit', value: 1, xp: '2' },
+]
+
+// Init data filled in the wheel
+var data = og;
+
+// Different options for the wheel
+var options = {
+  "og": og,
+  "mixers": mixers,
+  "shots": shots,
+  "sirup": sirup
+};

```

Comment:
Would be very cool if these options were easily editable "semi permanently".
The "easy" solution without a database would probably be to define `og.txt` and then have an interface to write to and load from this file through the website. Doesn't need to be git-controlled but could be so for the first few options at least. But one might consider this a new feature.

Reply:
Resolved